quadproduction/OpenPype
# #387: bugfix: use workdir from OpenPype instead of hardcoded value

## Changelog Description
Get workdir from OpenPype (thanks to anatomy object) when converting windows to linux paths instead of using hardcoded values.

## Testing notes:
1. Open any Blender scene with modifiers or render nodes
2. Launch "Prepare scene and render with Deadline" addon
3. Check in temporary generated files for output paths for modifers and renders
